### ISSUE TITLE
Deprecated react-countdown-now package

### ISSUE THREAD

**Issue by hbh7** [PollBuddy/PollBuddy#194]
**Please describe what has to be done**

Deprecated react-countdown-now package, should see if it's even in use and if so, update, and if not, remove. 

**Additional context**

```
C:\Users\hbh7\Documents\GitHub\PollBuddy\PollBuddy-Server\frontend>npm install
npm WARN deprecated react-countdown-now@2.1.2: This package has been renamed to "react-countdown".
```

**Comment by schans2** [comment]
Ok, I've combed through all the frontend JS files and the only one I found that package in was the `question.js` component. Will look into its use and update it.

**Comment by hbh7** [comment]
Fixed in #240
